frontend/packages/kubevirt-plugin/integration-tests-cypress/view/selector.ts

```diff
@@ -63,3 +63,6 @@ export enum storageClass {
   dropdown = 'button[id="form-ds-sc-select"]',
   selectMenu = '.pf-c-select__menu',
 }
+
+// yaml page
+export const yamlCreate = 'button[data-test="save-changes"]';
```

Comment:
I think we should be consistent and include `Button` or `Btn` at the end of all button selectors. I don't care which one we use. I just think it needs to be consistent. I think the verb should go first in this one, like `createYamlButton` or `createYamlBtn`.